### ISSUE TITLE
Apps - "Category" filter

### ISSUE THREAD

**Issue by dasunhegoda** [openspp-project/openspp-registry#513]
**Business Requirement**
The system shall include a "Category" filter, such as "OpenSPP," that displays only the relevant modules within our platform.

**Additional Information**

- **Priority:** [Low]
- **Dependencies:** [Other tasks, modules, etc.]
- **Attachments:** [Screenshots, diagrams, etc.]

![Image](https://github.com/user-attachments/assets/5435c184-02cd-4219-8940-cb304b27f2cd)

**Comment by anthonymarkQA** [comment]
Findings: QA Passed

QA Farmer registry :
![image](https://github.com/user-attachments/assets/46590a66-5543-4278-b5bf-24f3b718a684)

QA SP-MIS :
![image](https://github.com/user-attachments/assets/e09a3ac1-2d4a-4501-abd6-be0d9162cb9a)

**Comment by anthonymarkQA** [comment]
Findings: Returned to DEV

In total there are 74 OPENSPP modules, but in OPENSPP category it only displays 61


![Image](https://github.com/user-attachments/assets/ab18934e-0850-40fb-a8d6-c2a141233a00)

**Comment by anthonymarkQA** [comment]
Findings:  Returned to DEV

Searching "spp_" gives 74 results
But clicking at OpenSPP category, it only displays 61 results,
therefore there are 13 modules that are not categorized under "OpenSPP" yet.


Expected: All 74 OpenSPP results will display upon clicking the OpenSPP Category.

**Comment by kneckinator** [comment]
PR has been approved, but the issue is in "In review". What's the next step?

**Comment by dasunhegoda** [comment]
In review means PR to be reviewed.
Then, it should be in the release branch for the QA release.
Once the QA release is done, the ticket should be in QA.

**Comment by kneckinator** [comment]
> In review means PR to be reviewed. 

As I wrote above, the PR has already been approved and thus reviewed.

**Comment by dasunhegoda** [comment]
Once reviewed, it should be merged into the release branch for QA to test.

**Comment by kneckinator** [comment]
Re-opening issue, as work is remaining.

**Comment by anthonymarkQA** [comment]
FIndings:  not working for QA farmer registry

As @reichie020212  investigated thoroughly, it is 75 for expected number of OpenSPP modules , 76 for laos
currently it is only displaying 73 modules, 

`OpenSPP Custom Filter `and` OpenSPP Custom Filter UI ` are missing

**Comment by jannahadlaon** [comment]
FIndings: not working for **QA SP-MIS**
As per the comment above, 75 is the expected number of OpenSPP modules , 76 for laos and currently only 73 modules are shown.
`OpenSPP Custom Filter` and `OpenSPP Custom Filter UI` are missing
![image](https://github.com/user-attachments/assets/b3e0a0d1-90c6-47c3-bcaf-a32ce2fab0ed)

**Comment by anthonymarkQA** [comment]
Findings: QA Passed for QA Farmer registry base 

![Image](https://github.com/user-attachments/assets/5a92cd67-bdb6-4c3a-8a72-93b6c020d5f7)

**Comment by jannahadlaon** [comment]
Findings: QA Passed for SP-MIS base
![image](https://github.com/user-attachments/assets/c0994a56-4b85-4bdc-9ca3-687603745788)
